src/core/biblio.js

```diff
@@ -93,24 +93,61 @@ export function wireReference(rawRef, target = "_blank") {
   `;
 }
 
-export function stringifyReference(ref) {
+export function stringifyReference(ref,style) {
   if (typeof ref === "string") return ref;
-  let output = `<cite>${ref.title}</cite>`;
-  if (ref.href) {
-    output = `<a href="${ref.href}">${output}</a>. `;
+  let output = "";
+  switch (style) {
+  	case "APA": 
+  		// Author, A. (Year, Month Date of Publication). Article title. Retrieved from URL
```

Comment:
The logic is good, but instead of building out the output incrementally, would it be possible to gather all the parts, then have a template string at the end where all the parts are put together?

```JS
  const authors = ...;
  const date = ...;
  return `${authors} ${date} <cite>${title}</cite> ${URL}`;
```